Examples/Covasim/environment.py

```diff
@@ -25,18 +26,16 @@ def __init__(self, data = {
         "start_date": "2020-04-01",
         "end_date": "2021-04-01",
         "episode_num": "10",
-        "model_name":"SEIRDV",
```

Comment:
I'd lean towards CovasimLangata (as it's the implementation of the Langata Model this time done with Covasim)